windows/rtl-sdr.h

```diff
@@ -0,0 +1,366 @@
+/*
```

Comment:
Is there any recommended way of including these files (i.e. by using submodules)?